source/_integrations/fortigate.markdown

```diff
@@ -12,6 +12,10 @@ ha_domain: fortigate
 
 This is a FortiGate presence sensor based on device detection of the FortiGate API
 
+<div class='note warning'>
+This integration will be deprecated. Please use the [FortiOS](/integrations/fortios) integration.
+</div>
```

Comment:
This is now broken on our website as the HTML & Markdown context isn't switched.
At least one new line is needed after switching between HTML & Markdown.

So after the opening div and before the closing div an empty line needed to be put.